.gitignore

```diff
@@ -38,3 +38,5 @@ nosetests.xml
 .mr.developer.cfg
 .project
 .pydevproject
+
+pyspectral/version.py
```

Comment:
I'm not sure I understand. Is this generated on the fly so that it doesn't need to be registered by git?

Reply:
Correct. It is dynamically generated during install (ex. `pip install -e .`). I'm not sure if it would ever be a problem to do it except in the case that someone tries to run pyspectral without actually installing it in their environment. In that case instead of getting an error that they haven't installed the package they would get an incorrect version number. Not the end of the world in most cases. Regardless, it doesn't *need* to be tracked by git so it shouldn't be.